scale_build/image/bootstrap.py

```diff
@@ -0,0 +1,42 @@
+import os
+import shutil
+
+from scale_build.utils.run import run
+from scale_build.utils.paths import CD_DIR, CHROOT_BASEDIR, PKG_DIR, RELEASE_DIR, TMPFS
+
+from .utils import PACKAGE_PATH
+
+
+def setup_chroot_basedir(bootstrapdir_obj, logger):
+    shutil.rmtree(CHROOT_BASEDIR, ignore_errors=True)
```

Comment:
What errors are you ignoring?

We expect `CHROOT_BASEDIR` to be non-existent after this method terminates successfully. Will it always be so?